plugins/scaffolder-backend-module-gitlab/src/actions/gitlabMergeRequest.ts

```diff
@@ -78,7 +109,9 @@ export const createPublishGitlabMergeRequestAction = (options: {
           repoUrl: {
             type: 'string',
             title: 'Repository Location',
-            description: `Accepts the format 'gitlab.com?repo=project_name&owner=group_name' where 'project_name' is the repository name and 'group_name' is a group or username`,
+            description: `\
```

Comment:
Hmm, not sure about these `\`s in the text here, I wonder if you can either use `\n` or just have breaks in the backtick blocks instead. It's a little strange to rely on the markdown rendering component to strip these properly.

Reply:
AFAIK, and I just tested from `node` CLI, these are just escaping the newlines in the `` string so that the resulting string is unchanged from the original, only more readable.

Reply:
TIL